### DISCUSSION TITLE
How do we register for service worker updates with serwist?

### DISCUSSION THREAD

**Discussion by tony19** [serwist/serwist#62] [Q&A]
In the [docs for `next-pwa`](https://ducanh-next-pwa.vercel.app/docs/next-pwa/getting-started#tips), you point to this example:

> [You may want to ask user to reload when a new service worker is installed](https://github.com/DuCanhGH/next-pwa/tree/master/examples/lifecycle/components/PWALifecycle.tsx#L20-L40)

How would we do the same thing with serwist?

**Comment by DuCanhGH** [comment]
@tony19 `@serwist/window`'s APIs were barely changed from `workbox-window`'s ones. In this example, just replace `window.workbox` with `window.serwist`.

**Reply by tony19** [reply]
Thanks, that worked!
